e2e/src/utils.ts

```diff
@@ -407,8 +413,14 @@ export const utils = {
       },
     ]),
 
-  deleteTempFolder: () => {
+  resetTempFolder: () => {
     rmSync(`${testAssetDir}/temp`, { recursive: true, force: true });
+    mkdirSync(`${testAssetDir}/temp`, { recursive: true });
+  },
```

Comment:
This is not what we want as this will keep dummy files. That results in changes in git which I personally find very annoying

Reply:
This deletes all the files in the directory.

Reply:
Yeah nvm I'm stupid.